### PR TITLE
gives clumsy to waldo

### PR DESCRIPTION
**Pull request by ynot01**
# General Documentation

### Intent of your Pull Request

makes it so waldo is clumsy

### Why is this change good for the game?

wally shouldnt be rushing cap office and armory getting an arsenal and then giving the middle finger to the fugitives kinda shtewpid innit hes already got a wizard spell and hes supposed to be hiding or something

# Changelog

:cl:  
tweak: Gave clumsy trait to Waldo
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
He just doesnt need the nerf. Hes already sitting pretty comfortably balance wise. dont forget hes flashable and literally cannot get flash protection due to the no drop glasses. You can simply dress as an assistant and flash him when he comes near since he cant murderbone and cant get flash protection

**Comment by SomeguyManperson** [comment]
amount of times i've seen waldo be used as a "pseudo antag but without the pseudo" is at least more than one which is bad like come on it's not funny having waldo kill the entire security department because "they might turn me in" the first time much less the like tenth

File: code/modules/antagonists/fugitive/fugitive.dm [hunk 1]
```diff
@@ -41,6 +41,7 @@
 			to_chat(owner, "<B>Hi, Friends!</B>")
 			to_chat(owner, "<B>My name is Waldo. I'm just setting off on a galaxywide hike. You can come too. All you have to do is find me.</B>")
 			to_chat(owner, "<B>By the way, I'm not traveling on my own. wherever I go, there are lots of other characters for you to spot. First find the people trying to capture me! They're somewhere around the station!</B>")
+			owner.dna.add_mutation(CLOWNMUT)
```
Comment (SomeguyManperson) [inline code/modules/antagonists/fugitive/fugitive.dm]:
```suggestion
			owner.current.dna.add_mutation(CLOWNMUT)
```

File: code/modules/antagonists/fugitive/fugitive.dm [hunk 2]
```diff
@@ -41,6 +41,7 @@
 			to_chat(owner, "<B>Hi, Friends!</B>")
 			to_chat(owner, "<B>My name is Waldo. I'm just setting off on a galaxywide hike. You can come too. All you have to do is find me.</B>")
 			to_chat(owner, "<B>By the way, I'm not traveling on my own. wherever I go, there are lots of other characters for you to spot. First find the people trying to capture me! They're somewhere around the station!</B>")
+			owner.current.dna.add_mutation(CLOWNMUT)
```
Comment (SomeguyManperson) [inline code/modules/antagonists/fugitive/fugitive.dm]:
```suggestion
			var/mob/living/carbon/human/H = owner.current
			H.dna.add_mutation(CLOWNMUT)
```
should actually work

**Comment by Marmio64** [comment]
> He just doesnt need the nerf. Hes already sitting pretty comfortably balance wise. dont forget hes flashable and literally cannot get flash protection due to the no drop glasses. You can simply dress as an assistant and flash him when he comes near since he cant murderbone and cant get flash protection

le welder mask, welder gasmask, almost any hardsuit, and shielded eyes has arrived

**Comment by Ratvar** [comment]
If flash issues are an excuse for this shit, why not remove flash issues, alexkar

**Comment by UselessTheremin** [comment]
he is already
A. pacifist
B. has irremovable clothing.
as theos stated, he has literally no way to EVA or get flash protection without surgery or space adapt. this is a dumb balance change, he's already weak enough.

**Comment by UselessTheremin** [comment]
> > He just doesnt need the nerf. Hes already sitting pretty comfortably balance wise. dont forget hes flashable and literally cannot get flash protection due to the no drop glasses. You can simply dress as an assistant and flash him when he comes near since he cant murderbone and cant get flash protection
> 
> le welder mask, welder gasmask, almost any hardsuit, and shielded eyes has arrived

cant wear hardsuits, welder gasmask is something researched, cant wear welder mask, shielded eyes literally require surgery

**Comment by ynot01** [comment]
> A. pacifist

lmao what no if he was pacifist this wouldnt be an issue

**Comment by UselessTheremin** [comment]
> > A. pacifist
> 
> lmao what no if he was pacifist this wouldnt be an issue

Waldo has pacifism. I played him just the other day and he literally has pacifism

**Comment by SomeguyManperson** [comment]
I don't see anything pointing towards waldo being given pacifism

**Comment by MegaEmpirical** [comment]
I've never heard of Waldo being a pacifist, I have however seen them loot the hell out of the armory.

They're supposed to be a hide and seek mid round, I think this is a really solid change and in line with the idea of Waldo. Wouldn't say no to pacifism either.

**Comment by JamieD1** [comment]
Alright, but how can they fight the hunters?

**Comment by MegaEmpirical** [comment]
They don't actually have to fight them necessarily, that aside: without pascist they can use their hands, melee weapons, and thrown objects.

**Comment by Hopekz** [comment]
> You can simply dress as an assistant and flash him when he comes near since he cant murderbone and cant get flash protection

It's not murderbone if someone is validhunting you

**Review by redmoogle — APPROVED** [review]
Code is fine

**Comment by JamieD1** [comment]
HD: Not required currently, may look at if it becomes an issue in future.

**Comment by MegaEmpirical** [comment]
Saaaaad

**Pull request closed without merging**